pkgs/test/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 1.13.0-dev
+## 1.13.0
```

Comment:
Can you copy in the changes from the `test_api` changelog since this is the one users will see?